### PR TITLE
reword sessions to security in personal settings

### PR DESCRIPTION
**Pull request by icewind1991**
From https://github.com/nextcloud/server/pull/5166#discussion_r119912910

### CODE REVIEW

File: settings/templates/personal.php [hunk 1]
```diff
@@ -334,9 +334,9 @@
 	<?php }?>
 </div>
 
-<div id="sessions" class="section">
-	<h2><?php p($l->t('Sessions'));?></h2>
-	<p class="settings-hint hidden-when-empty"><?php p($l->t('Web, desktop and mobile clients currently logged in to your account.'));?></p>
+<div id="security" class="section">
+	<h2><?php p($l->t('Security'));?></h2>
+	<p class="settings-hint hidden-when-empty"><?php p($l->t('Web, desktop mobile clients and app specific passwords that currently have access to your account.'));?></p>
```
Comment (nickvergessen) [inline settings/templates/personal.php]: missing `,` between desktop and mobile?

Comment (icewind1991) [inline settings/templates/personal.php]: fixed

**Review by MorrisJobke — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Comment by MariusBluem** [comment]
Please backport.

**Comment by MorrisJobke** [comment]
Please remove the backporten label once the backporten PR is opened ;)

**Comment by jancborchardt** [comment]
Cool! :) Simple wording fix which makes things more understandable.

**Pull request merged**